lib/src/screens/dashboard/desktop_widgets/desktop_wallet_selection_dropdown.dart

```diff
@@ -26,84 +29,75 @@ class _DesktopWalletSelectionDropDownState extends State<DesktopWalletSelectionD
   final havenIcon = Image.asset('assets/images/haven_logo.png', height: 24, width: 24);
   final nonWalletTypeIcon = Image.asset('assets/images/close.png', height: 24, width: 24);
 
-  final double tileHeight = 60;
-
   Flushbar<void>? _progressBar;
 
   @override
   Widget build(BuildContext context) {
     final themeData = Theme.of(context);
-    return DropdownButton<WalletListItem>(
-      items: widget.walletListViewModel.wallets
-          .map((wallet) => DropdownMenuItem(
+    final dropDownItems = [
+      ...widget.walletListViewModel.wallets
+          .map((wallet) => DesktopDropdownItem(
+                isSelected: wallet.isCurrent,
                 child: ConstrainedBox(
                   constraints: BoxConstraints(maxWidth: 500),
-                  child: walletListItemTile(wallet),
+                  child: DropDownItemWidget(
+                      title: wallet.name,
+                      image: wallet.isEnabled ? _imageFor(type: wallet.type) : nonWalletTypeIcon),
                 ),
-                value: wallet,
+                onSelected: () => _onSelectedWallet(wallet),
               ))
           .toList(),
-      onChanged: (selectedWallet) async {
-        if (selectedWallet!.isCurrent || !selectedWallet.isEnabled) {
-          return;
-        }
+      DesktopDropdownItem(
+        onSelected: () => _navigateToCreateWallet(),
+        child: DropDownItemWidget(title: S.of(context).create_new, image: nonWalletTypeIcon),
+      ),
+      DesktopDropdownItem(
+        onSelected: () => _navigateToRestoreWallet(),
+        child: DropDownItemWidget(title: S.of(context).restore_wallet, image: nonWalletTypeIcon),
```

Comment:
please use the create and restore Icons

<img width="199" alt="Screen Shot 2023-02-22 at 4 15 35 PM" src="https://user-images.githubusercontent.com/31170694/220647243-46211bd5-2500-43cf-8963-6b720a4ca001.png">